src/components/App/index.js

```diff
@@ -48,6 +49,7 @@ const App = ({
       <hr />
       <section id="rules" className="Rules">
         <h1 className="App-Section-Title">Rules</h1>
+        <Rules/>
```

Comment:
hmm, I did it differently with sections. Do u think I should change it r it doesn't matter if they are done differently?